### ISSUE TITLE
Hide pokemon number from embedded sprite image

### ISSUE THREAD

**Issue by GeorgeCiesinski** [GeorgeCiesinski/poke-guesser-bot#4]
# Describe the new feature
Currently, when the sprite is generated and embedded from a reply, the link is parsed into a name for the pasted image. Opening the sprite in the browser reveals the name including the Pokemon's number, which can be used to extrapolate the pokemon. This spriteUrl name should be hidden.

## Planning
This should be resolved by downloading the image from the spriteUrl into a local generic png like `pokemonSprite.png`, then embedded from the local file into the embed message. 

Dominik recommended to use the fs tool in JS. 

Further research required.

**Comment by Wissididom** [comment]
The Discord-API only allows http and https.
```UnhandledPromiseRejectionWarning: DiscordAPIError: Invalid Form Body
embed.image.url: Scheme "./pokemon.png" is not supported. Scheme must be one of ('http', 'https').```

**Comment by GeorgeCiesinski** [comment]
Resolved in Pull Request #7
